### PR TITLE
adding recipe for r-jmbayes2via conda_r_skeleton_helper

### PR DESCRIPTION
**Pull request by dillonroach**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-jmbayes2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    - r-rcpparmadillo
+    - r-coda
+    - r-ggplot2
+    - r-gridextra
+    - r-matrixstats
+    - r-nlme
+    - r-parallelly
+    - r-survival
+  run:
+    - r-base
```
Comment (dbast) [inline recipes/r-jmbayes2/meta.yaml:49]:
```suggestion
    - libblas
    - r-base
```

File: recipes/r-jmbayes2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    - {{ compiler('c') }}          # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
```
Comment (dbast) [inline recipes/r-jmbayes2/meta.yaml:35]:
```suggestion
    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
    - llvm-openmp                       # [osx]
    - libgomp                                # [linux]
```

File: recipes/r-jmbayes2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    - r-rcpparmadillo
+    - r-coda
+    - r-ggplot2
+    - r-gridextra
+    - r-matrixstats
+    - r-nlme
+    - r-parallelly
+    - r-survival
+  run:
+    - libblas
```
Comment (dbast) [inline recipes/r-jmbayes2/meta.yaml:51]:
```suggestion
    - r-survival
    - libblas
  run:
```

File: recipes/r-jmbayes2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-coda
+    - r-ggplot2
+    - r-gridextra
+    - r-matrixstats
+    - r-nlme
+    - r-parallelly
+    - r-survival
+    - libblas
```
Comment (dbast) [inline recipes/r-jmbayes2/meta.yaml:50]:
```suggestion
    - libblas
    - liblapack
```

File: recipes/r-jmbayes2/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/JMbayes2_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/JMbayes2/JMbayes2_{{ version }}.tar.gz
+  sha256: 27b51c6b075bfc90421809e369f317dcdad5b7cf509a925c29c93b7ef2b8ae61
+
+build:
+  merge_build_host: True           # [win]
+  number: 0
```
Comment (isuruf) [inline recipes/r-jmbayes2/meta.yaml:17]:
```suggestion
  number: 0
  skip: True  # [r_base == "4.1"]
```

**Comment by dillonroach** [comment]
@dbast @isuruf Thanks for the help on this one - much appreciated! Good to go for review

**Comment by dillonroach** [comment]
@conda-forge/help-r ready for review!

**Pull request merged**